src/lib/api/directionality.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Inject, Injectable, InjectionToken, Optional} from '@angular/core';
+
+export type Direction = 'ltr'|'rtl';
+
+/**
+ * Injection token used to inject the document into Directionality.
+ * This is used so that the value can be faked in tests.
+ */
+export const DIR_DOCUMENT = new InjectionToken<Document>('fx-dir-document');
+
+/**
+ * The directionality (LTR / RTL) context for the application (or a subtree of it).
+ * Exposes the current direction and a stream of direction changes.
+ */
+@Injectable()
+export class Directionality {
+  /** The current 'ltr' or 'rtl' value. */
+  readonly value: Direction = 'ltr';
+
+  constructor(@Optional() @Inject(DIR_DOCUMENT) _document?: any) {
```

Comment:
Is there a reason you're not importing the `DOCUMENT` token from `angular/common`? This is why Travis is failing on prerender, it doesn't have access to the global browser `document` object